spec/features/email_spec.rb

```diff
@@ -40,6 +40,20 @@
     expect(Event.find(@event.id).acceptances_have_been_sent).to be(true)
   end
 
+  scenario "logged in as Organizer after sending an acceptance Email to the applicants of an event which has no rejected applications the event application status for sending rejections gets locked" do
+    @event = FactoryGirl.create(:event_with_accepted_applications,
+                                rejected_application_letters_count: 0,
```

Comment:
the other test don't have it
(same reasoning, phase those not effect email sending once your on the email site)

unless we find a convention for this (see the discussion alex started on our slack), I would leave it like thos